dask/dataframe/groupby.py

```diff
@@ -1787,7 +1805,7 @@ def aggregate(self, arg, split_every=None, split_out=1, shuffle=None):
                 chunk=_groupby_apply_funcs,
                 chunk_kwargs=dict(
                     funcs=chunk_funcs,
-                    sort=self.sort,
+                    sort=False,
```

Comment:
Note to self (but please do correct if I'm wrong): There is no "need" to sort intermediate aggregations, and the sort=True version **should** be slower in most/all cases. The fact that we have observed regressions when doing this is likely a Pandas bug (missed fast-path opportunity for categorical columns).

Reply:
This is all correct as far as I understand it